app/code/Magento/Catalog/Test/Mftf/Test/AdminVerifyStorefrontPropertiesTabStillActiveAfterSaveContinueEditTest.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminVerifyStorefrontPropertiesTabStillActiveAfterSaveContinueEditTest">
+        <annotations>
+            <features value="Catalog"/>
+            <stories value="Verify tab active after save form product attribute"/>
+            <title value="Activate Tab no longer works on legacy tabs/forms"/>
+            <description value="Admin should be able visible previous tab active after save product attribute"/>
+            <severity value="AVERAGE"/>
+            <testCaseId value="MC-38658"/>
+            <group value="Catalog"/>
+        </annotations>
+
+        <before>
+            <actionGroup ref="AdminLoginActionGroup" stepKey="LoginAsAdmin"/>
+        </before>
+
+        <after>
+            <!-- Delete created attribute -->
+            <actionGroup ref="DeleteProductAttributeActionGroup" stepKey="deleteAttribute">
+                <argument name="ProductAttribute" value="productTextEditorAttribute"/>
+            </actionGroup>
+
+            <actionGroup ref="AdminLogoutActionGroup" stepKey="logoutAsAdmin"/>
+            <!-- Reindex invalidated indices after product attribute has been created/deleted -->
+            <magentoCron groups="index" stepKey="reindexInvalidatedIndices"/>
+        </after>
+
+        <!-- Go to Stores > Product, click "Add New Attribute" -->
+        <actionGroup ref="AdminOpenProductAttributePageActionGroup" stepKey="openProductAttributePage"/>
+        <click selector="{{AdminProductAttributeGridSection.createNewAttributeBtn}}" stepKey="createNewAttribute"/>
+        <waitForPageLoad stepKey="waitForNewAttributeFormLoad"/>
```

Comment:
COuld you please also move it to action group ? Tests should have only Action Group according to Magento MFTF Best Practice.

Reply:
i have updated following mftf best practice